.travis.yml

```diff
@@ -5,7 +5,8 @@ os:
     - windows
 julia:
     - 1.0
-    - 1.4
+    - 1.3
```

Comment:
This change seems odd. But regardless, I think testing on LTS (1.0) and current release is sufficient.

Reply:
The reason for this change is that -- in my perception -- many things changed between 1.0 and 1.3 and between 1.3 and 1.5, but not so much between 1.3 and 1.4. This observation is, of course, subjective. I can revert this change, if you like, while the PR is still open.

Reply:
I see - I'm fine with that. Should add 1.5 though since it's currently the stable release